client/app.jsx

```diff
@@ -44,10 +45,27 @@ class App extends React.Component {
       })
   }
 
+  generateDates() {
+    let today = new Date;
+    let month = today.getMonth();
+    let year = today.getFullYear();
+    let dayCount = [...Array(367).keys()].slice(1);
```

Comment:
What is this doing?

Reply:
So this actually just generates every date and month for the next 367 days. Each month is passed as props to make a page and each date a div on the page.